CHANGELOG.md

```diff
@@ -8,11 +8,13 @@ All notable changes to this project starting with the 0.6.0 release will be docu
 - New command `site redis disable` to disable Redis caching. (#813)
 - New command `site solr enable` to enable Solr indexing. (#814)
 - New command `site solr disable` to disable Solr indexing. (#814)
+- Added `--email=<email>` argument to `auth login` to retireve saved machine tokens. (#825)
```

Comment:
Can we standardize on the existing email input? E.g.:

`terminus auth login [email]` 

Would first check for a matching token, and re-use if valid. If not, prompt for PW.

This will help us gradually get people transitioned over to tokens. Maybe in the next release we can add a notice for people (post artwork) that logging in via passwords is old news and they should get a token from the dashboard.